### ISSUE TITLE
Sceptre will say stack not found when stack is in ROLLBACK_COMPLETE

### ISSUE THREAD

**Issue by matrinox** [Sceptre/sceptre#411]
```sh
$ sceptre --var-file vars/dev.yaml delete-stack environment stack
[2018-06-19 10:28:09] - sceptre.stack - environment/stack - Deleting stack
[2018-06-19 10:28:21] - sceptre.connection_manager - Request limit exceeded, pausing...
[2018-06-19 10:28:27] - sceptre.connection_manager - Request limit exceeded, pausing...
[2018-06-19 10:28:32] - sceptre.stack - environment/stack does not exist.

$ aws cloudformation --describe-stacks --stack-name environment/stack
{
    "Stacks": [
        {
            "StackId": "arn:aws:cloudformation:us-east-1:***:stack/environment-stack/***",
            "StackName": "environment-stack",
            "Description": "Deploys environment-stack\n",
            "Parameters": [
                ...
            ],
            "CreationTime": "2018-06-18T12:14:55.183Z",
            "DeletionTime": "2018-06-18T12:15:28.596Z",
            "RollbackConfiguration": {},
            "StackStatus": "ROLLBACK_COMPLETE",
            "DisableRollback": false,
            "NotificationARNs": [],
            "Capabilities": [
                "CAPABILITY_IAM",
                "CAPABILITY_NAMED_IAM"
            ],
            "Tags": [
                ...
            ],
            "EnableTerminationProtection": false
        }
    ]
}
```

Technically none of the resources are active but it'd be better if they can do cleanup duty and remove the stacks.

**Comment by ngfgrant** [comment]
Good spot. I think we can probably handle this better.

**Comment by zaro0508** [comment]
I believe this is fixed, I cannot reproduce in sceptre version 2.7.1
